cabal-install/src/Distribution/Client/CmdListBin.hs

```diff
@@ -133,7 +133,14 @@ listbinAction flags@NixStyleFlags{..} args globalFlags = do
 
     case binfiles of
         []     -> die' verbosity "No target found"
-        [exe] -> info normal exe
+        [exe]  -> putStr $ withOutputMarker verbosity exe
```

Comment:
Why not `putStrLn`?

Reply:
Ouch, all tests had EOF at the end, so this wasn't visible.

Reply:
Yeah, `putStrLn` would be better in the normal operation.  I guess I got pulled towards `putStr` because the `withOutputMarker` puts the EOL character there: https://github.com/haskell/cabal/blob/a78d77b7edd758b9cda3617111445397ac4e2673/Cabal/src/Distribution/Simple/Utils.hs#L622-L625